### PR TITLE
Match csv download

### LINKED ISSUE #91: download matches as a csv

### PR DESCRIPTION
**Pull request by angelazhang043**
Right now the make matches button is still not an admin action- we wanted to get the full team opinion about how to integrate that button with the rest of the admin view.

### CODE REVIEW

File: match/admin.py [hunk 1]
```diff
@@ -3,35 +3,41 @@
...
+    readonly_fields = ("user_id", "match_id", "name", "last_name","match_first_name", "match_last_name")
+    actions = ['export_matches']
     
 
     def get_urls(self):
         urls = super().get_urls()
         my_urls = [
             path('makematches/', self.make_matches),
-            path('download/', self.export_matches),
+            # path('download/', self.export_matches),
```
Comment (ezronis) [inline match/admin.py:28]: remove comment

Comment (ezronis) [inline match/admin.py]: remove comment

File: match/admin.py [hunk 2]
```diff
@@ -3,35 +3,41 @@
...
 
     def make_matches(self, request):
-        r = requests.get('http://127.0.0.1:8000/survey/json/5')
+        r = requests.get('http://127.0.0.1:8000/survey/json/4')
         r.json()
         print(os.getcwd())
         # x = requests.post('http://127.0.0.1:8000/api/v1/mlalgorithmstatuses', json = r.json())
        
-        mock_json = pd.read_json('/Users/student/MentorApp/match/mock.json', orient = 'records')
+        mock_json = pd.read_json('/mnt/c/Users/angel/OneDrive/Documents/WITProject/MentorApp/match/mock.json', orient = 'records')
```
Comment (ezronis) [inline match/admin.py]: remove the absolute path and make it a local path

File: match/admin.py [hunk 3]
```diff
@@ -3,35 +3,41 @@
...
         print(mock_json)
         dict_json = mock_json.to_dict('records')
-        # mock_json = x.to_json
```
Comment (ezronis) [inline match/admin.py:34]: remove comment

File: mentorapp/settings.py [hunk 4]
```diff
@@ -115,8 +118,8 @@
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
             'NAME': 'mentorapp',
-            'USER': 'name',
-            'PASSWORD': '',
+            'USER': 'angelazhang',
```
Comment (ezronis) [inline mentorapp/settings.py]: revert back to name

File: mentorapp/settings.py [hunk 5]
```diff
@@ -115,8 +118,8 @@
...
+            'PASSWORD': 'password',
```
Comment (ezronis) [inline mentorapp/settings.py]: revert pack to empty string

File: some.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+User ID,Username,User's First Name,User's Last Name,Match ID,Match's First Name,Match's Last Name
```
Comment (ezronis) [inline some.csv]: remove this file from the commit

**Review by ezronis — CHANGES_REQUESTED** [review]

File: match/admin.py (hunk 3 above)
Comment (angelazhang043) [reply match/admin.py:34]: I don't think this is still in the branch?

**Pull request merged**